modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/NearTxResultHandler.java

```diff
@@ -108,6 +109,9 @@ public static GridNearTxEnlistResponse createResponse(GridDhtTxEnlistFuture fut)
         try {
             cctx.io().send(nearNodeId, res, cctx.ioPolicy());
         }
+        catch (ClusterTopologyCheckedException e) {
```

Comment:
Why this change?

Reply:
Code speed-up increase probability of this case. 
It was a problem before, but very rarely.